gradio/components/chatbot.py

```diff
@@ -54,6 +56,8 @@ def __init__(
         """
         Parameters:
             value: Default value to show in chatbot. If callable, the function will be called whenever the app loads to set the initial value of the component.
+            enable_latex: If True, will render LaTeX expressions enclosed in $$ delimiters or custom delimiters passed in `latex_delimiters` parameter. True by default.
+            latex_delimiters: A list of dicts of the form {open_delimiter, close_delimiter, display} that will be used to render LaTeX expressions. By default this is set to `[{ left: "$$", right: "$$", display: true }]`, so LaTeX expressions enclosed in $$ delimiters will be rendered.
```

Comment:
I think it would be explain this needs to be a subset of KaTeX delimeters and maybe link to their docs